master/usage/assigning-ip-addresses-topology.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Assigning IP addresses based on topology
+---
+
+## Why this is useful
+
+{{site.prodname}} can be configured to use specific IP pools for different topological areas.
+For example, you may want workloads in a particular rack, zone, or region to receive addresses from the same IP pool.
+This may be desirable either to reduce the number of routes required in the network or to meet requirements imposed by an external firewall device or policy.
+This guide walks through an example of how to set this up.
+
+## How this works
+At a high level, this feature is operated by the setting node labels and then selecting those
+nodes via node selector on your IP pool resources. Note that there are a variety of other ways
+to specify IP address assignment behavior that can be found in the
+[cni-plugin configuration reference document]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration).
+Each approach has its own use-case but this particular one takes on the lowest priority
```

Comment:
In general, i.e. can always be replaced with just "for example" so many editors might advise this. In this case, it would really be a lot better to make the parenthetical into a new sentence that starts with "For example," (delete i.e. as you can't start a sentence with that), and remove the parentheses. While you're at it, for completeness, don't forget to close the preceding sentence with a period.